pkg/cmd/render/render.go

```diff
@@ -621,18 +621,16 @@ func writeManifests(outputDir, templateDir string, templateData interface{}) err
 	return nil
 }
 
-func isSingleStackIPv6(serviceCIDRs []string) (bool, error) {
+func preferIPv6(serviceCIDRs []string) (bool, error) {
```

Comment:
would you mind adding a quick unit test for this func specifically? just testing a couple of input/output pairs

Reply:
Yep, will do.

Reply:
awesome, thanks again.